testing/common.test.js

```diff
@@ -0,0 +1,72 @@
+import glob from 'glob';
+import { ClientFunction } from 'testcafe';
+import { join } from 'path';
+import { existsSync, readFileSync } from 'fs';
+import { compareScreenshot } from './helpers/screenshot-comparer';
+
+const execCode = ClientFunction((code) => {
+  // eslint-disable-next-line no-eval
+  const result = eval(code);
+  if (result && typeof result.then === 'function') {
+    return Promise.race([result, new Promise((resolve) => setTimeout(30000, resolve))]);
+  }
+
+  return Promise.resolve();
+});
+
+const execTestCafeCode = (t, code) => {
+  // eslint-disable-next-line no-eval
+  const testCafeFunction = eval(code);
+  return testCafeFunction(t);
+};
+
+fixture`Getting Started`
+  .beforeEach((t) => t.resizeWindow(1000, 800))
+  .clientScripts([
+    { module: 'mockdate' },
+  ]);
+
+const getDemoPaths = (platform) => glob.sync(`JSDemos/Demos/**/${platform}`);
+
+['jQuery'/* , 'React', 'Vue', 'Angular' */].forEach((approach) => {
+  const demoPaths = getDemoPaths(approach);
+
+  demoPaths.forEach((demoPath) => {
+    const testParts = demoPath.split('/');
+    const widgetName = testParts[2];
+    const demoName = testParts[3];
+    const testName = `${widgetName}-${demoName}`;
+
+    const preTestCodePath = join(demoPath, '../pre-test-code.js');
+    const testCodePath = join(demoPath, '../test-code.js');
+    const testCafeTestCodePath = join(demoPath, '../testcafe-test-code.js');
+
+    // if (widgetName !== 'DateBox' || demoName !== 'Overview') {
```

Comment:
Forgotten comments?

Reply:
No, this comment can be uncommented and modified to debug specific test

Reply:
It does not look obvious. How do you think anybody will figure it out?